File: Backend/src/main/java/edu/icet/demo/dto/Order.java
package edu.icet.demo.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.ToString;

@Data
@AllArgsConstructor
@NoArgsConstructor
@ToString
public class Order {
    private Integer id;
    private String frameName;
    private String size;
    private String material;
    private String emailAddress;
    private String address;
    private String phoneNumber;
    private String photoUrl;
    private String comment;
    private String orderState;
}

File: frontEnd/src/model/Order.ts
export class Order{

    address :String;
    comments : String;
    emailAddress : String;
    phoneNumber : String;
    photoUrl :String;
    orderState :String;
    itemCode :String;
    price: String;








constructor(address :String,
    comments : String,
    emailAddress : String,
    phoneNumber : String,
    photoUrl :String,
    orderState :String,
    itemCode :String,
    price: String

){
this.itemCode =itemCode;

this.emailAddress = emailAddress;
this.address =address;
this.phoneNumber = phoneNumber;
this.photoUrl =photoUrl;
this.comments = comments;
this.price=price;
this.orderState=orderState ;


}


}